config.js

```diff
@@ -4,7 +4,12 @@ const AppConfig = {
     APPLICATION_PORT: process.env.PORT || 5000,
     CONNECTION_URL: 'mongodb+srv://qcsdevs:4JneurUiw1NfAjcj@clusterone-ecp1h.mongodb.net/events-app?retryWrites=true&w=majority',
     DATABASE_NAME: 'events-app',
-    SECRET: 'SUPER_secret' // TODO: possibly embedded as environment variable so it's not directly visible
+    SECRET: 'SUPER_SECRET_SRBIJA_SRBIMA', // TODO: possibly embedded as environment variable so it's not directly visible,
+    REFRESH_TOKEN_SECRET: "SUPER_SECRET_KOSOVO_I_METOHIJA",
```

Comment:
Single quotations should be used throughout project as coding standard/practice.